.github/workflows/changelog.yml

```diff
@@ -16,6 +16,6 @@ jobs:
       - name: Grep CHANGES.md for PR number
         if: contains(github.event.pull_request.labels.*.name, 'skip news') != true
         run: |
-          grep -Pz "PR( |\n\s*)#${{ github.event.pull_request.number }}[^0-9]" CHANGES.md || \
-          (echo "Please add 'PR #${{ github.event.pull_request.number }}' change line to CHANGES.md" && \
+          grep -Pz "\(( |\n\s*)#${{ github.event.pull_request.number }}[^0-9]\)" CHANGES.md || \
```

Comment:
This regex won't match now because it's expecting `({whitespaceOrNewline}#123{someRandomNonNumericChar}). Here's two that work:
1. `\((\n\s*)?#${{ github.event.pull_request.number }}(\n\s*)?\)` - does support having the brackets on separate lines
2. `\(#${{ github.event.pull_request.number }}\)` - does NOT support having the brackets on separate lines

I have no idea whether prettier will (or how it will) break `(#123)` when it violates the line length limit so IDK which regex is better. Your call, although the first one seems safer?

Reply:
Let's go with the first suggestion here and land this.

Reply:
Will fix now.